.circleci/config.yml

```diff
@@ -69,13 +69,13 @@ linux_default: &linux_default
         docker exec -e CIRCLE_JOB=${CIRCLE_JOB} -u jenkins $(cat .docker_pid) .circleci/test.sh
 
 jobs:
-  py2-gcc7-ubuntu16.04:
+  py3.6-clang7-ubuntu16.04:
```

Comment:
So, we are dropping support for python 2.x?

Reply:
For this pull request, we are only changing the CI job that verifies the integration between ONNX and PyTorch and onnxruntime backends to use Python3. This makes senses because both backends have dropped py2 support already.
There are still py2 other CI jobs (running on Travis) that checks ONNX is py2 compatible. Given the developers of the Python language started dropping support for py2 from 01/02/2020, I think there should be discussions in the corresponding SIG to decide on the timeline of dropping py2 support universally in ONNX.